nextcord/guild.py

```diff
@@ -3079,6 +3080,41 @@ async def ban(
 
         await self._state.http.ban(user.id, self.id, delete_message_seconds, reason=reason)
 
+    async def bulk_ban(self, users: List[Snowflake], *, delete_message_seconds: int = 0) -> BulkBan:
+        """|coro|
+
+        Bans users from a guild in bulk. This has a limit of 200 users
+
+        The list of users must meet the :class:`abc.Snowflake` abc.
```

Comment:
```suggestion
```
This part is pretty self-explanatory, and most users would provide the `User` objects anyway.